lib/constructor.js

```diff
@@ -226,6 +226,7 @@ const Sharp = function (input, options) {
     negateAlpha: true,
     medianSize: 0,
     blurSigma: 0,
+    precision: 'integer',
```

Comment:
If I'm not mistaken this is the current default if not precision is passed to vips gaussblur